README.md

```diff
@@ -18,6 +18,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [World Satoshi Summit](https://www.worldsatoshisummit.com/) | 12-13 May | New Delhi | Bring all the students, entrepreneurs, blockchain enthusiasts, blockchain developers and influencers under one roof for a seamless exchange of ideas. |
 | [Rootconf 2018](https://rootconf.in/2018/) | 10-11 May | NIMHANS Convention Centre, Bangalore  | Conference on DevOps and IT Infrastructure |
 | [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem. |
+| [Global Azure Bootcamp](http://bengaluru.azurebootcamp.net/) | 21 Apr | ABB Ability Innovation Center, Bengaluru | Global Azure Bootcamp 2018 @ Bengaluru is a full day conference about the Microsoft Azure Cloud Computing Platform organized by Bengaluru Azure User Group collaborating with BDotnet, BItPro, OSSBangalore and BangaloreaOS Communities. It is free and open for everybody to join. |
```

Comment:
It's probably me :) but the website doesn't load for me and is continuously stuck in the "loading" phase.

Can you send a screenshot of what it looks like just to validate the details? Thanks!

Reply:
Never mind, it finally loaded. :)

Can you update the description to just this (taken from the website):

```
Global Azure Bootcamp is a full day conference about developing Cloud Computing Applications for Microsoft Azure.
```

I'd like it to be short but convey the message too.

Reply:
Oh! I did not see this at all. @nikhita 
Let me know I can still update this.

Reply:
@aravindputrevu Feel free to update it. But funnily enough, the website doesn't seem to work for me again. Is it just me or can you not access it as well?

Reply:
I think they removed it because as the event is over.

Reply:
Ouch.

@aravindputrevu I'll close this PR since the event is over and the website is non-functional.

Thanks for submitting the PR though, really appreciate it!